doc/source/user_guide/krylov.rst

```diff
@@ -0,0 +1,156 @@
+
+#. Via Mechanical APDL commands.
+#. Via macros, a feature described in *Frequency-Sweep Harmonic Analysis via the Krylov Method* in the *Structural Analysis Guide*.
+#. PyMAPDL implementation of the macros, a feature described here.
+
+Assumptions
+-----------
+The following assumptions are made in the Krylov solution obtained using this method :
+
+* The stiffness, mass, and damping matrices are assumed to be constant (independent of frequency).
+
+* The external load vector is linearly ramped over frequency. Ramping assumes that the frequency at 
+  which the Krylov subspace is built is in the middle of the frequency range. If you wish to apply 
+  stepped loading, there is an option to specify that in the inputs for the 
+  :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` method.
+
+
+Krylov : PyMAPDL implementation
+===============================
+The PyMAPDL implementation gives the user customization and flexibility in terms of accessing
+the subspace vectors and reduced solutions in the python format for defining user defined routines.
+
+If you do not require customization, you can use the simpler using APDL commands to solve a harmonic analysis 
+with the Krylov method.
+ 
+The procedure described here implements an analysis identical to that defined by the macros.
+Details on the theory and equations describing the Krylov method can be found in the works of Puri [1]_ and Eser [2]_ .
+
+The exposure in PyMAPDL follows the same the same theory as the apdl macro and has the following methods:
+
+* :func:`KrylovSolver.krygensub() <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` Creates the Krylov subspace. 
+* :func:`KrylovSolver.krysolve() <ansys.mapdl.core.krylov.KrylovSolver.krysolve>` Solves the reduced system of equations.
+* :func:`KrylovSolver.kryexpand() <ansys.mapdl.core.krylov.KrylovSolver.kryexpand>` Expands the Krylov subspace.
+
+.. warning:: These methods must be run consecutively.
+
+Usage
+=====
+
+Generate the full file (``.full``) for Krylov method using Mechanical APDL:
+
+.. code:: py
+
+    >>> from ansys.mapdl.core import launch_mapdl
+    
+    >>> mapdl = launch_mapdl()
+    >>> mapdl.prep7()
+
+	# Generate the FEA model (mesh, constraints, loads, etc.)
+  # ...
+
+    >>> mapdl.run("/SOLU")
+    >>> mapdl.antype("HARMIC")  # HARMONIC ANALYSIS
+    >>> mapdl.hropt("KRYLOV")
+    >>> mapdl.eqslv("SPARSE")
+    >>> mapdl.harfrq(0,1000)  # Set beginning and ending frequency
+    >>> mapdl.nsubst(100)  # Set the number of frequency increments
+    >>> mapdl.wrfull(1)  # GENERATE .FULL FILE AND STOP
+    >>> mapdl.solve()
+    >>> mapdl.finish()
+
+Create Instance of the Krylov Class
+-----------------------------------
+
+.. code:: py
+    
+    >>> mk = mapdl.krylov
+
+Call the :func:`krygensub <ansys.mapdl.core.krylov.KrylovSolver.krygensub>` method which creates the Krylov subspace:
+Build a subspace of Size / Dimension 10 and at a frequency of 500 Hz.
```

Comment:
```suggestion
Build a subspace of size/dimension 10 and at a frequency of 500 Hz.
```

Reply:
Is the "and" needed inline 90 at "and at a ...". It seems awkward.

Reply:
changed to "Build a subspace of size/dimension 10 at a frequency of 500 Hz."